wf/en-us/Content/Topics/Workflows/function-reference/Text/text_match.htm

```diff
@@ -455,7 +455,7 @@ <h2>Example</h2>
                                                                 <tr class="TableStyle-standard-horizonal-borders-only-Body-Body1">
                                                                     <td class="TableStyle-standard-horizonal-borders-only-BodyE-Column1-Body1"><code>hello</code>
                                                                     </td>
-                                                                    <td class="TableStyle-standard-horizonal-borders-only-BodyE-Column1-Body1">1</td>
+                                                                    <td class="TableStyle-standard-horizonal-borders-only-BodyE-Column1-Body1">-1</td>

```

Comment:
This repo is only for output files - so I've added this to the appropriate source file and it will be updated with the rest of the WF docs this week. 

I'll close this PR as fixed elsewhere. For future changes, ping me on Slack and I'll show you the doc update process. 😄